contexts/credentials/v2

```diff
@@ -5,6 +5,28 @@
 
     "id": "@id",
     "type": "@type",
+    
+    "kid": {
+      "@id": "https://www.iana.org/assignments/jwt#kid",
+      "@type": "@id"
+    },
+    "iss": {
+      "@id": "https://www.iana.org/assignments/jwt#iss",
+      "@type": "@id"
+    },
+    "sub": {
+      "@id": "https://www.iana.org/assignments/jwt#sub",
+      "@type": "@id"
+    },
+    "jku": {
+      "@id": "https://www.iana.org/assignments/jwt#jku",
+      "@type": "@id"
+    },
+    "x5u": {
+      "@id": "https://www.iana.org/assignments/jwt#x5u",
```

Comment:
This doesn't exist.  Perhaps it should be a jose url?  I see the jose page duplicates lots of params?  And in both cases the HTML has no anchors at all.  Are these the canonical form that community uses for params?

Reply:
Actually, it seems that none of the proposed ID-s exist, not only "x5u".

Adding URL-s with broken fragments is a no-no in the case of normative W3C specifications. If we consider the context file's _content_ to be normative (which seems to be the case) then this applies to these URL-s as well. To make it clear: the publication process may reject this publication when it comes to CR/PR/Rec publishing.

Reply:
In order to fix the fragment identifiers that do exist, I expect that the JOSE WG would need to agree to update the JOSE registry to add the fragment identifiers. An alternative that might work is pointing directly to the specification, such as this one for "iss": https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.1 -- although, that's definitely not ideal.

Changing the fragment identifiers in the JOSE registry is probably the best path forward.

Reply:
Before I make this change, @davidlehn @iherman if I change this from:

https://www.iana.org/assignments/jwt/#iss (for all related fragments)

to

https://www.iana.org/assignments/jose/#iss (for all related fragments)

Will it resolve your concerns?

@iherman how would URLs in the normative v2 context block publication? 

Would the same apply to w3id.org URLs changing the underlying anchors?

Reply:
> Before I make this change, @davidlehn @iherman if I change this from:
>
> https://www.iana.org/assignments/jwt/#iss (for all related fragments)
>
> to
>
> https://www.iana.org/assignments/jose/#iss (for all related fragments)
>
> Will it resolve your concerns?

As far as i can see, it won't. The problem is identical: the target page does not have an `@id` value corresponding to the fragment in the URL.

> @iherman how would URLs in the normative v2 context block publication?

Because the context's _**content**_ is now normative, the publication rule applies: all URLs must resolve, and unresolved fragment ID-s are not o.k. This means that whoever would check these rules, whether it is a machine or one of the persons in the team checking the specification, would flag this.

> Would the same apply to w3id.org URLs changing the underlying anchors?

Yes. The general review of the security vocabulary is still waiting for us and yes, we are dreadfully behind schedule in my view...

Reply:
@iherman what happens if a context contains a URN? 

For example:  https://datatracker.ietf.org/doc/rfc9162/

`urn:ietf:params:trans:error:endBeforeStart`

Would publication be blocked?